gradle/defaults.gradle

```diff
@@ -19,13 +19,15 @@ allprojects {
   apply plugin: 'base'
 
   group "org.apache"
+  
+  def lucenePrereleaseBuild = '5'
 
   // Repositories to fetch dependencies from.
   repositories {
     mavenCentral()
     maven {
-      name "ApacheSnapshots"
```

Comment:
I think this could be made more friendly, Uwe. You could declare the "maven" repository conditionally based on a flag (or even auto-detected version pattern). An alternative is to declare two different repositories (maven and prerelease) and then push publications to the right one but this complicates things a bit.

Reply:
Sorry, disregard. I thought it's related to pushing artifacts from lucene side.

Reply:
In short: This PR is just for "consuming" the already published artifacts from a directory on an arbitrary webserver.

Reply:
sorry @dweiss have seen you latest message too late!